### PR TITLE
contrib/shotman: new package (0.4.5)

### PR DESCRIPTION
**Pull request by mhmdanas**

### CODE REVIEW

File: contrib/shotman/template.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+options = ["!check"]
+
+
+def post_build(self):
+    self.do("bmake", "shotman.1")
+
+
+def post_install(self):
+    self.install_license("LICENCE.md")
+    self.install_man("shotman.1")
```
Comment (mhmdanas) [inline contrib/shotman/template.py:27]: The package also has completions, but they're generated by running a binary, which won't work in cross. Not sure what to do about that.

Comment (nekopsykose) [reply contrib/shotman/template.py:27]: you could make it !cross like sbctl

Comment (mhmdanas) [reply contrib/shotman/template.py:27]: Is it worth making it !cross just for completions? We could generate them only on native builds, but idk

Comment (q66) [reply contrib/shotman/template.py:27]: !cross is fine

File: contrib/shotman/template.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+pkgname = "shotman"
+pkgver = "0.4.5"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo", "pkgconf", "scdoc"]
+makedepends = ["libxkbcommon-devel", "rust-std"]
+depends = ["slurp"]
+pkgdesc = "Uncompromising screenshot GUI for Wayland"
```
Comment (q66) [inline contrib/shotman/template.py:8]: Screenshot GUI for Wayland

**Pull request merged**